lib/iris/fileformats/grib/_grib_cf_map.py

```diff
@@ -190,33 +193,37 @@
     CFName('lagrangian_tendency_of_air_pressure', None, 'Pa s-1'): G2Param(2, 0, 2, 8),
     CFName('land_area_fraction', None, '1'): G2Param(2, 2, 0, 0),
     CFName('land_binary_mask', None, '1'): G2Param(2, 2, 0, 0),
+    CFName('land_binary_mask', None, '1'): G2Param(2, 2, 0, 0),
     CFName('liquid_water_content_of_surface_snow', None, 'kg m-2'): G2Param(2, 0, 1, 13),
     CFName('low_type_cloud_area_fraction', None, '%'): G2Param(2, 0, 6, 3),
     CFName('medium_type_cloud_area_fraction', None, '%'): G2Param(2, 0, 6, 4),
     CFName('moisture_content_of_soil_layer', None, 'kg m-2'): G2Param(2, 2, 0, 22),
     CFName('precipitation_amount', None, 'kg m-2'): G2Param(2, 0, 1, 49),
+    CFName('precipitation_amount', None, 'kg m-2'): G2Param(2, 0, 1, 52),
```

Comment:
This CFName (`precipitation_amount`) appears to be associated with two `G2Param`s (on this line and the line above).